src/raftstore/store/region_snapshot.rs

```diff
@@ -532,32 +532,32 @@ mod tests {
         let mut iter = Cursor::new(snap.iter(IterOption::default()), ScanMode::Mixed);
         assert!(
             !iter
-                .reverse_seek(&Key::from_encoded(b"a2".to_vec()), &mut statistics)
+                .reverse_seek(&Key::from_encoded_slice(b"a2"), &mut statistics)
```

Comment:
Off topic.. is Key::from_encoded correct in this test case? It seems to be "from_raw" 🤣

Reply:
oh, I will clean these. if it should be `from_raw()` I will fix it in other PR.